js/LocationSampler.ts

```diff
@@ -59,6 +59,8 @@ export default class LocationSampler {
                 if (!err) {
                     const data = JSON.stringify({ battery_after: level,
                                                 battery_before: this.batteryLevelStart,
+                                                samplerate: this.interval,
+                                                measurementname: this.measurementName,
```

Comment:
If you switch these lines so they are sorted alphabetically, TSLint will not fail once #6 is merged into master.